### DISCUSSION TITLE
UI bug on rewards tab of explorer

### DISCUSSION THREAD

**Discussion by jim-counter** [subquery/subql#991] [Frontier TestNet - Bug Reporting]
Small issue - the UI is telling me I have unclaimed rewards but they are all in a claimed state.

![image](https://user-images.githubusercontent.com/11335994/167291954-5ae96b08-3d60-48a0-b466-73cdb6f68ebe.png)

**Comment by mzxyz** [comment]
@CaiYiLiang
